google/ggrc-core
# GGRC-7460 Inconsistency for display values in Rich text fields

# Issue description
Inconsistency for display values in Rich text fields.

# Steps to test the changes 
1. Log in ggrc app and open any object with GCA rich text, e.g. Requirement.
2. Type text to ASSESSMENT PROCEDURE as bold.
3. Type text to GCA rich text as bold.
4. Open Requirement in tree view and set ASSESSMENT PROCEDURE and GCA rich text columns.
5. Look at the tree view.

Actual Result: Text in ASSESSMENT PROCEDURE is not bold, GCA rich text is bold.
Expected Result: Unify the behavior of rich text fields.

_Note: After clarification with BA, it was found out that it was necessary for custom user formatting to be displayed._

# Solution description 
Display custom user formatting.

# Sanity checklist 

# PR Review checklist
